library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -2902,6 +2905,23 @@ std::string getSolutionNameFromAlgoIndex(rocblaslt_handle handle, const rocblasl
     return solution->solutionName;
 }
 
+rocblaslt_status isDeviceSupported()
```

Comment:
@jichangjichang Is this code thread-safe?

Reply:
@jithunnair-amd  yes